mautic/mautic
# My activity on user profile

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR adds a new tab on the user account page to show a history of the last 150 events related to the account. This is helpful in cases like "someone logged into the account and changed/deleted something" or "my campaign is gone and idk why" (customer support).

The goal here is to have a humanized description that clients can also understand.

It features:
- Translation strings so the history is internationalized for each object in their description
- Names of updated/created things such as campaigns or forms
- A clean design that allows anyone to print this info (Save as PDF)


![image](https://github.com/user-attachments/assets/4c6dcb3d-84b9-41f7-98c7-d217ce189db1)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open your Account > My activity and see some
3. Modify things across Mautic to generate more logs and come back there to see again
